src/main/java/com/softvision/books/services/domain/BookFilter.java

```diff
@@ -0,0 +1,60 @@
+package com.softvision.books.services.domain;
+
+import java.util.Objects;
+
+public class BookFilter {
```

Comment:
May not need to implement filter as finding books by title can be implemented by utilizing the built in query methods like below:

Page<Book> findByTitleContaining(String title, Pageable pageable)